src/script/script.cpp

```diff
@@ -70,7 +70,7 @@ const char* GetOpName(opcodetype opcode)
 
     // splice ops
     case OP_CAT                    : return "OP_CAT";
-    case OP_SUBSTR                 : return "OP_SUBSTR";
```

Comment:
can you explain why this is removed? don't see that upstream

Reply:
>OP_SPLIT replaces OP_SUBSTR and uses it's opcode.

Specs: https://github.com/bitcoincashorg/bitcoincash.org/blob/master/spec/may-2018-reenabled-opcodes.md#op_split

Implementation: https://reviews.bitcoinabc.org/rABCce813ee28ee8a08bbe458958803836358fa4c372